.github/workflows/bench.yml

```diff
@@ -5,11 +5,14 @@ on:
     branches:
       - dev
   workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/bench.yml'
```

Comment:
Thanks for help fixing benchmark!
Could you keep it in dev branch only?

Reply:
Ok @wusyong I've removed the new trigger and re-pushed the PR.

Anyway, it wouldn't have published the benchmarks, as they are published only on `dev` branch: https://github.com/tauri-apps/wry/blob/293907224e935f64ea7d344acb422379f8a23b1d/.github/workflows/bench.yml#L73-L74